app/adapters/web-extension.js

```diff
@@ -55,6 +61,23 @@ export default BasicAdapter.extend({
     this.onResourceAdded();
   },
 
+  _setThemeColors() {
+    const rootElement = document.querySelector('html');
+    let theme;
+
+    if (chrome.devtools.panels.themeName === 'dark') {
```

Comment:
How do browsers other than Chrome handle this statement?

Reply:
I'm not sure, but it must be okay because we are doing several `chrome.devtools` things in several methods in this file.

Reply:
@nummi @rwwagner90 I did fail to verify this. Thank you for the wakeup call on this. I have now verified Opera and Firefox. Opera handles this statement without issues. It works in Firefox but the add-on requires a manual reload. The Devtool extensions in both Opera and Chrome seem to be automatically forced to reload when the settings are adjusted. 

I will work on fixing the issue in Firefox.

Reply:
I just noticed that the React-Devtools requires the same manual reload in order to respond to theme changes within Firefox's devtool settings. Firefox might be a bit tricky to deal with regarding this.

Reply:
@pbishop16 is Firefox okay if you start with normal or dark themes? It just has issues if you try to switch themes?

Reply:
@rwwagner90 Yeah, it sure is. It displays light or dark accordingly.

Reply:
@pbishop16 I think that is fine then. We should do whatever the React ones do.